lib/test_suite.py

```diff
@@ -185,7 +185,7 @@ def collect_tests(self):
         return self.tests
 
     def get_fragile_list(self):
-        return self.fragile['tests'].keys()
+        return list(self.fragile['tests'])
```

Comment:
I would just replace `self.get_fragile_list()` in `stable_tests()` and `fragile_tests()` with `self.fragile['tests']`. So no extra list will be created. However, okay, it is up to you, because `_list` is in the function name.

However, as I see, the change is not necessary: everything works good with `.keys()` on Python 3. So it is better to don't change it.

Reply:
> However, as I see, the change is not necessary: everything works good with .keys() on Python 3. 

Just reverted change